PixiEditor/Helpers/RelayCommand.cs

```diff
@@ -3,6 +3,7 @@
 
 namespace PixiEditor.Helpers
 {
+    [Obsolete("Use GalaSoft.MvvmLight.CommandWpf.RelayCommand instead")]

```

Comment:
Ummm, why did you mark it as obsolete? I told you we should use our RelayCommand

Reply:
We actually said on discord that we were gonna make PixiEditors RelayCommand obselote when I made that commit

And I forgot to remove it again after you said we're gonna use PixiEditor because it supports generics too now